internal/action/config.go

```diff
@@ -34,10 +35,10 @@ func (s *Action) Config(c *cli.Context) error {
 		return exit.Error(exit.Usage, nil, "Usage: %s config key value", s.Name)
 	}
 
-	// need to initialize sub-stores so we can update their configs.
-	// special case: we can always update the root config.
-	if err := s.IsInitialized(c); err != nil && store != "" {
-		return err
+	// sub-stores need to have been initialized so we can update their local configs.
+	// special case: we can always update the global config. NB: IsInitialized initializes the store if nil.
+	if inited, err := s.Store.IsInitialized(ctx); err != nil || store != "" && !inited {
```

Comment:
Could you add brackets to the if condition? My brain is struggling to remember if || or && is evaluated first.